src/components/Details/Details.tsx

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+import React from "react";
+import "../Details/Details.css"
 
-interface DetailsProps {}
+interface MarketProps {
+  markets: {
+    id: number;
+    distanceFromZip: number;
+    marketName: string
+  }[],
+  marketDetails: {
+    street: string,
+    city: string,
+    state: string,
+    zip: string,
+    schedule: {
+      dayOfWeek: string,
+      time: string,
+      season: string
+    }[],
+    products: string[],
+    mapsLink: string
+  }[],
+  id: string;
+}
+
+const Details: React.FC<MarketProps> = ({ id, markets, marketDetails }) => {
+
+  const marketMatch = markets.filter(market => market.id === Number(id));
+  const detailsMatch = marketDetails.filter(market => { 
+    let match = market.mapsLink.split('%22')[1].includes(marketMatch[0].marketName.split(' ')[0]);
+    console.log(marketMatch[0].marketName)
+    return match
+  });
+  const nameMatch = marketMatch[0].marketName;
+  const detail = detailsMatch[0];
+  const openSeason = `Season: ${detail.schedule[0].season}`;
+  const daysAndTimes = `Open: ${detail.schedule[0].dayOfWeek} ${detail.schedule[0].time}`;
+  const productList = detail.products.map(prod => {
+    return (
+    <div className='list'>
+      <ul>
+        <li> {prod}, </li>
```

Comment:
Is this going to render the last product with a trailing ","?

Reply:
Comma removed. I have tried within the .map  {prod.indexOf(prod) === detail.products.length - 1 && <li>propWithoutComma<>} as well as a ternary.